src/components/Header.tsx

```
@@ -1,20 +1,77 @@
+              </button>
+            </DropdownMenuTrigger>
+            <DropdownMenuContent className="bg-white text-black">
+              <DropdownMenuItem>
+                <Link to="/perfil">Meu Perfil</Link>
+              </DropdownMenuItem>
+              <DropdownMenuItem>
+                <Link to="/meus-projetos">Meus Projetos</Link>
+              </DropdownMenuItem>
+              <DropdownMenuItem>
+                Logout
+              </DropdownMenuItem>
```

ESLint

[error] Replace `⏎················Logout⏎··············` with `Logout`

(prettier/prettier)